build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = '1.4.10'
+    ext.kotlin_version = '1.4.21'
```

Comment:
I would recommend to take a look at this link before upgrading to version 1.4.21 of kotlin.

https://kotlinlang.org/docs/reference/whatsnew1420.html#kotlin-android-extensions